java/vector/src/test/java/org/apache/arrow/vector/ipc/ITTestIPCWithLargeArrowBuffers.java

```diff
@@ -122,6 +122,53 @@ private void testWriteLargeArrowData(boolean streamMode) throws IOException {
     assertTrue(new File(FILE_NAME).exists());
   }
 
+  private void testWriteLargeArrowDataThruBuilderPattern(boolean streamMode) throws IOException {
```

Comment:
I don't think we need to duplicate every test using the builder pattern, especially if the builder is implemented without having to add new constructors.